docs/translations/zh/dot-net/index.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Just spot-checked this file - this is also incorrect. Please ensure you're downloading & using the Contents v1.1 files.